torch/csrc/jit/codegen/cuda/parser.cpp

```diff
@@ -1473,6 +1532,26 @@ bool insertProfileIValue(ProfilingRecord* pr, Node* node, size_t offset) {
   if (node->input(offset)->node()->kind() == prim::Constant) {
     return false;
   }
+  
+  static auto dropout_schema =
+      getOperatorForLiteral(
+          "aten::dropout(Tensor input, float p, bool train) -> Tensor")
+          ->schema();
+  static auto native_dropout_schema =
+      getOperatorForLiteral(
+          "aten::native_dropout(Tensor input, float p, float scale, bool train) -> (Tensor, Tensor)")
+          ->schema();
+  if (node->matches(dropout_schema) || node->matches(native_dropout_schema)) {
+    switch (offset) {
+      // argument 2: Is training?
+      case 2:
```

Comment:
[BUG] the offset here is inconsistent between `native_dropout` (3) and `dropout` (2)

Reply:
Nice catch, I forgot I changed the signature.